Library/Homebrew/cask/config.rb

```diff
@@ -182,6 +182,17 @@ def merge(other)
       self.class.new(explicit: other.explicit.merge(explicit))
     end
 
+    sig { returns(String) }
+    def explicit_s
+      explicit.map do |key, value|
+        if key == :languages
```

Comment:
Would be nice if there was some way of linking this to the other use(s) of `:languages`/`language` so that they don't get out of sync. Worst case, just a comment pointing to the other case.

Reply:
The man page says to use [`--language`](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/language.md#installation) to specify one or more languages, but the `explicit` hash in `cask.config` uses the key [`languages`](https://github.com/Homebrew/brew/blob/4ae8dfe43faface13c164e0f45185905abca98b1/Library/Homebrew/cask/config.rb#L59) (if that's what you mean by "other uses"). Adding comments to point this out.